theme/placeholder.scss

```diff
@@ -2,8 +2,15 @@
 // For licensing, see LICENSE.md or http://ckeditor.com/license
 
 .ck-placeholder::before {
+	position: absolute;
+	top: 0;
+	left: 0;
 	content: attr( data-placeholder );
 	cursor: text;
 	color: #c2c2c2;
 	pointer-events: none; // See ckeditor/ckeditor5#469.
 }
+
+.ck-placeholder {
```

Comment:
If we'll do anything dangerous with these styles I'd like to have very strong reasons to do so. In fact, some bugs are easier to live with (or hack) than such CSS rules. Therefore, as I expressed in https://github.com/ckeditor/ckeditor5-engine/pull/1032#issuecomment-317274915, I'd like to understand if there are really so strong reasons to go this way.